src/drivers/heater/heater.cpp

```diff
@@ -88,35 +72,7 @@ int Heater::custom_command(int argc, char *argv[])
 		return PX4_ERROR;
 	}
 
-	const char *arg_v = argv[0];
-
-	// Display/Set the heater controller period value (usec).
-	if (strcmp(arg_v, "controller_period") == 0) {
-		return get_instance()->controller_period(argv);
-	}
-
-	// Display/Set the heater driver integrator gain value.
-	if (strcmp(arg_v, "integrator") == 0) {
-		return get_instance()->integrator(argv);
-	}
-
-	// Display/Set the heater driver proportional gain value.
-	if (strcmp(arg_v, "proportional") == 0) {
-		return get_instance()->proportional(argv);
-	}
-
-	// Display the id of the sensor we are controlling temperature on.
-	if (strcmp(arg_v, "sensor_id") == 0) {
-		return get_instance()->sensor_id();
-	}
-
-	// Displays/Set the current IMU temperature setpoint.
-	if (strcmp(arg_v, "setpoint") == 0) {
-		return get_instance()->temperature_setpoint(argv);
-	}
-
-	get_instance()->print_usage("Unrecognized command.");
-	return PX4_OK;
+	return print_usage("Unrecognized command.");
```

Comment:
Is the argument for removing these thaat to adjust gains you need to let the board cool anyways so you may as well reboot?